### PR TITLE
issuer processor config

### PR DESCRIPTION
**Pull request by ps2-controller**
]

### CODE REVIEW

File: openapi.json [hunk 1]
```diff
@@ -1406,6 +1406,37 @@
           }
         }
       }
+    },
+    "/organization/issuer_processors": {
```
Comment (kpamaran) [inline openapi.json:1410]: The actual route is /organization/issuer_processor, I also see /organization/payment_processor has the same issue as well.

Comment (ps2-controller) [reply openapi.json:1410]: Okay sounds good, that makes sense - we can upstream together

File: responses/ConfigureIssuerProcessors.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Payment Processor configuration for your organization",
```
Comment (kpamaran) [inline responses/ConfigureIssuerProcessors.json:2]: Change word payment to issuer

**Review by kpamaran — APPROVED** [review]

**Comment by kpamaran** [comment]
Created https://github.com/canopy1/canopyAPI/pull/483

**Pull request merged**